client/src/components/Form/Form.scss

```diff
@@ -161,6 +161,10 @@
     border-color: $input-focus-border-color;
     outline: none;
   }
+
+  &:hover {
+    cursor: not-allowed;
```

Comment:
This shouldn't be the cursor state.  It implies they're doing something wrong by mousing over it, though they may just want to select the text so they can copy-paste it which is perfectly fine.

Reply:
Updated